corehq/apps/case_search/xpath_functions/query_functions.py

```diff
@@ -12,6 +12,7 @@
 from corehq.apps.es.case_search import (
     case_property_geo_distance,
     case_property_query,
+    sounds_like_text_query,
```

Comment:
@Robert-Costello this is no longer defined in the PR for some reason

Reply:
Maybe that one was just me. Adding it back in.